nats/README.md

```diff
@@ -15,6 +15,13 @@ Current version is `0.1.0`.
 
 This object MUST NOT contain any properties. Its name is reserved for future use.
 
+##### Fixed Fields
+
+Field Name | Type | Description
+---|:---:|---
+<a name="serverBindingObjectPayload"></a>`payload` | string | Defines the payload data type. Can be either `string`, `binary` or `JSON` (default). https://github.com/nats-io/nats.ts#connect-options
```

Comment:
Reading the documentation [here](https://github.com/nats-io/nats.ts#connect-options), it says the default is `string` and this description says the default is `JSON`. Is it a typo or is there a reason for this?

Reply:
Typo yea :)